doc/examples/gatk/README.md

```diff
@@ -41,21 +55,23 @@ refSDF              dir                 22.57MiB
 $ cd ../../
 ```
 
-## Committing a sample
+## Committing a sample 
 
-Create a repositories for input `*.bam` files:
+Next, we're going to work on commiting the `*.bam` files for Mom. Let's stary by create a repositories for input `*.bam` files to go into:
 
 ```sh
 $ pachctl create-repo samples
 ```
 
-Add a `*.bam` file (along with it's index file) corresponding to a first sample (`mother`). Here we will assume that the files corresponding to each sample are committed to separate directories (e.g., `/mother`):
+Add a `*.bam` file (along with it's index file) corresponding to a first sample, in our case it's the `mother`. Here we will assume that the files corresponding to each sample are committed to separate directories (e.g., `/mother`). 
+
+**Important**: make sure to replace where it says `<yourcommitid>` with the commit-id created by `pachctl start-commit`
```

Comment:
Do we need to use commitIDs for some reason here or can we just use branch name `master`